WowPacketParser/Enums/Opcode.cs

```diff
@@ -2913,6 +2916,7 @@ public enum Opcode
         SMSG_TURN_IN_PETITION_RESULT,
         SMSG_TUTORIAL_FLAGS,
         SMSG_TWITTER_STATUS,
+        SMSG_QUERY_ARENA_TEAM_RESPONSE,
```

Comment:
Already exists as `SMSG_ARENA_TEAM_QUERY_RESPONSE` (from 3.3.5)

Reply:
No worries, finish reviewing it and I will fix those.